spec/forms/steps/income/income_savings_assets_form_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe Steps::Income::IncomeSavingsAssetsForm do
+  subject(:form) { described_class.new(arguments) }
+
+  let(:arguments) do
+    {
+      crime_application:,
+      has_frozen_income_or_assets:
+    }
+  end
+
+  let(:crime_application) { instance_double(CrimeApplication, income_details:) }
+  let(:income_details) { instance_double(IncomeDetails) }
+
+  let(:has_frozen_income_or_assets) { nil }
```

Comment:
Is this `has_frozen_income_or_assets = nil` declaration necessary as I think each context overrides it ?

Reply:
Thanks I'll check that